server_admin/topics/authentication/flows.adoc

```diff
@@ -342,29 +345,67 @@ claims= {
         }
 ----
 
-NOTE: To request the acr_values as text (such as `gold`) instead of a numeric value, you have to configure the mapping between the ACR and the LoA at the client level. For configuration see <<_mapping-acr-to-loa,ACR to LoA Mapping>>)
+The {project_name} javascript adapter has support for easy construct of this JSON and sending it in the login request.
+See link:{adapterguide_link_js_adapter}[Javascript adapter documentation] for more details.
+
+It is also possible to use simpler parameter `acr_values` instead of `claims` parameter to request particular levels as non-essential. This is mentioned
+in the OIDC specification.
+
+There is also possibility to configure default level for the particular client, which is used when parameter `acr_values` or parameter `claims` with the `acr` claim is not present.
```

Comment:
Change to 
"You can also configure the default..."
add "the" before "parameter" (both times)